scipy/special/bscript

```diff
@@ -6,6 +6,7 @@ def pre_build(context):
 
     context.tweak_extension("orthogonal_eval", use="NPYMATH")
     context.tweak_extension("lambertw", use="NPYMATH")
+    context.tweak_extension("spherical_bessel", use="NPYMATH")
```

Comment:
Probably not needed? The spherical functions are included in the _ufunc.so extension?